R/redisConnect.R

```diff
@@ -0,0 +1,21 @@
+#' redisConnect
+#'
+#' @import proto dplyr
```

Comment:
You're right, proto and dplyr are left overs from other code that I ripped out for the push.

You're also right that the optional password could be part of the constructor.  Unfortunately, I speak R better than Cpp, so it was faster/cleaner/more likely to result in success for me to write this part of the code in R.  I didn't think that dropping out to R for this part of the connection would have serious performance implications since connecting to the redis server would typically be a one-time affair.

Even if it is decided that it is mandatory to put the AUTH inside of the constructor, I would still encourage having an R function wrapper for connecting because it takes positional arguments and the way the function works seems more familiar to me from an class-impaired R-speaker standpoint than using the new().  But, mileage may vary.